pkg/controller/operandregistry/operandregistry_controller.go

```diff
@@ -121,28 +129,8 @@ func (r *ReconcileOperandRegistry) Reconcile(request reconcile.Request) (reconci
 
 	klog.V(1).Infof("Reconciling OperandRegistry %s", request.NamespacedName)
 
-	// Set the init status for OperandConfig instance
-	if !instance.InitRegistryStatus() {
-		if err := r.client.Update(context.TODO(), instance); err != nil {
-			return reconcile.Result{}, err
-		}
-	}
-
-	if err := r.updateRegistryOperatorsStatus(instance); err != nil {
-		return reconcile.Result{}, err
-	}
-	if len(instance.Status.OperatorsStatus) == 0 {
-		instance.UpdateRegistryPhase(operatorv1alpha1.RegistryInit)
-	} else {
-		instance.UpdateRegistryPhase(operatorv1alpha1.RegistryRunning)
-	}
-	if err := r.client.Status().Update(context.TODO(), instance); err != nil {
-		return reconcile.Result{}, err
-	}
-
 	// Set Finalizer for the OperandRegistry
-	added := instance.EnsureFinalizer()
```

Comment:
I wrote the condition in this way just want to make the code more readable.
I am ok with the change.
Could we enhance the comment to make it more clear?
For example
```
// // Set Finalizer for the OperandRegistry. If the OperandRegistry finalizer is added to the finalizer list, EnsureFinalizer() will return true. If the OperandRegistry finalizer already exists, EnsureFinalizer() will return false.
```

Reply:
Updated